pixi.js.d.ts

```diff
@@ -659,32 +665,32 @@ declare namespace PIXI {
         static TEMP_MATRIX: Matrix;
 
     }
-    export class ObservablePoint {
 
-        constructor(cb: () => any, scope?: any, x?: number, y?: number);
+    class PointLike {
```

Comment:
Sure, we would just have to move `set` and `equals` further down so PointLike just describes an object with `x` and `y` properties.